### ISSUE TITLE
Mana HUD Broken

### ISSUE THREAD

**Issue by Androues** [Mithion/Mana-And-Artifice#158]
Whenever I have the mana hud on all items in my inventory, jei, chests, etc. are invisible, all text is gone including waila and the settings/ escape menu. I have shaders installed but given that it is a GUI issue I assume it is unrelated, there are also other mods installed but none with any hud related things

**Comment by Mithion** [comment]
Optifine is the cause of that issue

**Comment by Androues** [comment]
Thank you for your help
Is there any setting I can turn off to fix it or is just having optifine installed the problem?

**Comment by Mithion** [comment]
I asked a friend of mine and she said:

"All you need is Optifine nowdays and throw the zip's from the shaders into the shader folder.
Some shaders are more compatible with mods that add particles/effects than others.

SUES is one of the more popular shaders but very incompatible with mods.
Black textures....

I use BSL (though modified), with Misa's texture pack (Which also matters a lot, because it's the texture packs that tell the shaders what to do):
https://lextermina7.wixsite.com/bslex"

She plays M&A without issue and with shaders.  Maybe you can take it from there?

**Comment by Mithion** [comment]
Link to Misa's as well https://www.curseforge.com/minecraft/texture-packs/misas-realistic/files

**Comment by Androues** [comment]
I've been using BSL, I'll try out the texture pack.
Thank you so much for the help

**Comment by jackg2527** [comment]
Hi there, I was having similar problems so after reading this string I did some testing. Sharing this in case it helps someone other than me.

Currently running:
1.16.5-forge-36.0.42
mana-and-artifice-1.3.0.2
geckolib-forge-1.16.5-3.0.14
curios-forge-1.16.5-4.0.4.0
OptiFine_1.16.5_HD_U_G7_MOD
Shaderpack = BSL_v7.2.01pre2

With no resource pack installed (default textures only): Right-click Codex for the first time, the mana HUD is visible but can't see inventory items or any text (including in the Codex), and all menu buttons are blank. Will call this the "GUI issue".

So I tried Misa_1.16.4c, and everything works fine. Right-click codex, cool sound, turns to night, no GUI issue.

Patrix and LBPR reload v4.5 behaved same as default. Once the GUI issue happens, disabling shaders OR switching to Misa's pack fixes it. Then I tried to figure out why Misa's worked but not the other resource packs.

Misa has optifine/lightmap folder with .png for all three dimensions. If I remove this folder, can reproduce GUI issues with Misa's pack. If I put optifine/lightmap folder into other resource packs (no other changes), they work fine! 

I dunno why having optifine/lightmap/world0.png fixes this, but hoping this info might help you or the optifine folks. Ars Magica has been one of my favorite mods since MC 1.6, happy that I can play your latest and satisfy my addiction to shaders :)

Cheers, and thanks for all the your awesome work!
